test/UnitTest/Microsoft.Test.AspNet.OData/Routing/ODataRoutingTest.cs

```diff
@@ -188,7 +188,9 @@ public ODataRoutingTest()
                     { "GET", "Products/Default.TopProductOfAll", "TopProductOfAll" },
                     { "GET", "Products/Default.TopProductOfAllByCity(city='any')", "TopProductOfAllByCity(any)" },
                     { "GET", "Products/Default.TopProductOfAllByCity(city='some%23hashes')", "TopProductOfAllByCity(some#hashes)" },
+#if NETFX
                     { "GET", "Products/Default.TopProductOfAllByCity(city='some%2fslashes')", "TopProductOfAllByCity(some/slashes)" },
```

Comment:
Is there something that NetCore doesn't support here?

Reply:
Thanks. I added a comment in the code.